bs/src/tablecloth.ml

```diff
@@ -276,8 +276,7 @@ module List = struct
 
   let insert_at = insertAt
 
-  let splitWhen ~(f : 'a -> bool) (l : 'a list) : ('a list * 'a list) option
-      =
+  let splitWhen ~(f : 'a -> bool) (l : 'a list) : ('a list * 'a list) option =
```

Comment:
For the styling, I'm using ocamlformat with these settings: https://github.com/darklang/tablecloth/issues/7

We should standardize on them at some point, but until then any format is fine.